rs/recovery/src/file_sync_helper.rs

```diff
@@ -85,12 +86,25 @@ pub fn rsync_with_retries(
             require_confirmation,
             key_file,
         ) {
-            Err(e) => warn!(logger, "Rsync failed: {:?}, retrying...", e),
+            Err(e) => {
+                warn!(logger, "Rsync failed: {:?}", e);
+                if auto_retry {
+                    // In non-interactive cases, we wait a short while
+                    // before re-trying rsync.
+                    info!(logger, "Retrying in 10 seconds...");
+                    std::thread::sleep(std::time::Duration::from_secs(10));
+                } else if !consent_given("Do you want to retry the  download for this node?") {
+                    return Err(RecoveryError::UnexpectedError(
```

Comment:
Maybe we should introduce a new error variant? To me `UnexpectedError` doesn't feel right here.

Not sure what a good name would be though

Reply:
What do you think about a simple `RsyncFailed` variant? Updated the code.